scripts/make_docs_locally.sh

```diff
@@ -4,22 +4,22 @@
 # python scripts/json_schema_to_md_doc.py
 # python scripts/generate_dialect_comparison_docs.py
 
-# NB not including docs/demos, as 02_Exploratory_analysis.ipynb may
-# cause process to hang on: Running 3 `page_markdown` events
-# only an issue with `serve` or `build` - `gh-deploy` works okay
-
-# UPDATE="TRUE"
-# if [[ $UPDATE == "TRUE" ]]
-# then
-#     rm -rf docs/demos/
-#     git clone https://github.com/moj-analytical-services/splink_demos.git docs/demos/
-# fi
+UPDATE="TRUE"
+if [[ $UPDATE == "TRUE" ]]
+then
+    rm -rf docs/demos/
+    git clone https://github.com/moj-analytical-services/splink_demos.git docs/demos/
+fi
 
 deactivate
 python3 -m venv docs-venv
 source docs-venv/bin/activate
 pip install --upgrade pip
 pip install -r scripts/docs-requirements.txt
 
+# manually preprocess include files as include-markdown plugin clashes with mknotebooks
+# make sure not to commit changes to files with inclusions!
+python3 scripts/preprocess_markdown_includes.py
+
```

Comment:
You need to ensure you've run `python3 scrips/generate_dialect_comparison_docs.py` prior to this step